src/toil/lib/docker.py

```diff
@@ -0,0 +1,255 @@
+import base64
+import logging
+import subprocess
+
+import os
+from bd2k.util.exceptions import require
+
+_logger = logging.getLogger(__name__)
+
+
+def dockerCall(job,
+               tool,
+               parameters=None,
+               workDir=None,
+               dockerParameters=None,
+               outfile=None,
+               defer=None):
+    """
+    Wrapper call to Docker. See docstring for _docker()
+    Throws CalledProcessorError if the Docker invocation returns a non-zero exit code
+
+    :param toil.Job.job job: The Job instance for the calling function.
+    :param str tool: Name of the Docker image to be used (e.g. quay.io/ucsc_cgl/samtools).
```

Comment:
I could also just append `:latest` to `quay.io/ucsc_cgl/samtools` as the example if you'd think that sufficient.